src/qos/logger.js

```diff
@@ -38,6 +38,9 @@ class Logger {
     let attributes = ''
     let tag
     if (tags) {
+      if (!tags.shard) {
+        tags.shard = Game.shard.name
+      }
```

Comment:
This isn't needed - the shard is automatically added to the tags on the server side.